resources/assets/coffee/forum/topic-ajax.coffee

```diff
@@ -42,6 +42,11 @@ $(document).on 'ajax:success', '.delete-post-link', (_event, data) ->
 
 $(document).on 'ajax:success', '.reply-post-link', (e, data) ->
   data += '\n'
+  if window.isDoublePost
```

Comment:
probably should intercept much earlier instead of waiting for server reply. `ajax:before` or `ajax:beforeSend`. [Reference](https://github.com/rails/jquery-ujs/wiki/ajax).